README.md

```diff
@@ -4,13 +4,15 @@ This repository contains the code needed to build the sidecar containers used fo
 
 The code in this repository should be located at ``$GOPATH/src/microsoft/confidential-sidecar-containers``.
 
-## Build Pipelines
+## Testing
 
-These build pipelines run nightly based off of images created from the main branch of this repo. They are used to test the latest changes to the sidecar containers.
+[![CI](https://github.com/microsoft/confidential-sidecar-containers/actions/workflows/ci.yml/badge.svg?branch=main&event=schedule)](https://github.com/microsoft/confidential-sidecar-containers/actions/workflows/ci.yml)
 
-- [![Test Key Release (Latest)](https://github.com/microsoft/confidential-aci-examples/actions/workflows/test_key_release_latest.yml/badge.svg)](https://github.com/microsoft/confidential-aci-examples/actions/workflows/test_key_release_latest.yml)
-- [![Test Encrypted Filesystem (Latest)](https://github.com/microsoft/confidential-aci-examples/actions/workflows/test_encrypted_filesystem_latest.yml/badge.svg)](https://github.com/microsoft/confidential-aci-examples/actions/workflows/test_encrypted_filesystem_latest.yml)
-- [![Test Attestation](https://github.com/microsoft/confidential-aci-examples/actions/workflows/test_attestation.yml/badge.svg)](https://github.com/microsoft/confidential-aci-examples/actions/workflows/test_attestation.yml)
+Each sidecar is tested under `./tests/<sidecar_name>`. Each directory is tested with [confidential-aci-testing](https://github.com/microsoft/confidential-aci-testing), and therefore contains:
```

Comment:
I would explain the purpose of files (or just not that they have to be updated if adding/changing tests) like cacitesting.env and the deployment folder.